vercel.json

```diff
@@ -0,0 +1,5 @@
+{
+  "buildCommand": "yarn build",
+  "installCommand": "dnf install -y $(cat chrome-dependencies.txt) && yarn install && npx browsers install chrome@131.0.6778.204 --path /vercel/.cache/puppeteer",
+  "framework": "nextjs"
+}
```

Comment:
@davemooreuws if anything is missing here that should be here based on the config provided via the dashboard let me know.

Reply:
All good, we just use the defaults in the vercel dashboard